pkg/controller/plan/kubevirt.go

```diff
@@ -239,6 +242,11 @@ func (r *KubeVirt) EnsureVM(vm *plan.VMStatus) (err error) {
 		err = liberr.Wrap(err)
 		return
 	}
+	//If the VM name is not valid according to DNS1123 labeling
+	//convention it will be automatically changed.
+	if len(k8svalidation.IsDNS1123Label(vm.Name)) > 0 {
```

Comment:
how about `if errs := IsDNS1123Label(label); len(errs) > 0 {`, I find it more readable than checking the length of a return value of a function that starts with 'Is'

Reply:
Agreed, changed according to your suggestion